index.html

```diff
@@ -2236,9 +2236,15 @@ <h2 id="docconformance">
             </td>
             <td>
               <p>
-                <a><strong class="nosupport">No `role`</strong></a>
+                If a `label` element is implicitly or explicitly associated with a 
+                <a href="data-html/https://html.spec.whatwg.org/multipage/forms.html#category-label">labelable element</a> then
+                <a><strong class="nosupport">no `role`</strong></a>
               </p>
-              <p class="addition"><a>Naming Prohibited</a></p>
+              <p>
+                Otherwise, if the `label` is not associted with an element then <a><strong>any `role`</strong></a> 
```

Comment:
```suggestion
                Otherwise, if the `label` is not associted with an element then <a><strong>any `role`</strong></a>, 
```

missing comma